drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -512,17 +740,22 @@ def _get_item_stats(self):
         Returns a tuple representing the number of correctly-placed items,
         and the total number of items that must be placed on the board (non-decoy items).
         """
-        all_items = self.data['items']
-        item_state = self._get_item_state()
+        required_items, __, correct_items = self._get_item_raw_stats()
 
-        required_items = [str(item['id']) for item in all_items if self._get_item_zones(item['id']) != []]
-        placed_items = [item for item in required_items if item in item_state]
-        correct_items = [item for item in placed_items if item_state[item]['correct']]
+        return len(correct_items), len(required_items)
+
+    def _get_item_raw_stats(self):
+        """
+        Returns a 3-tuple containing required, placed and correct items.
```

Comment:
Nit: Update docstring to include the Returns and descriptions as described here: 

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings